core/scripts/local/CalculateContractBytecode.js

```diff
@@ -0,0 +1,24 @@
+// This simple script tells you how big your contract byte code is and how much you have until you exceed
+// the current block limit as defined by EIP170.
+
+const argv = require("minimist")(process.argv.slice(), { string: ["contract"] });
+const contractName = argv.contract;
+
+if (!contractName) {
+  console.log("Please enter the contract name as a parameter as `--contract <name>`.");
+  return;
+}
+var child = require("child_process").exec("truffle compile");
+child.stdout.pipe(process.stdout);
+child.on("exit", function() {
+  console.log("finished compiling 🚀!");
+  console.log("loading", contractName + ".json");
+  let obj = require("./../../build/contracts/" + contractName + ".json");
+
+  const byteCodeSize = (obj.bytecode.length - 2) / 2;
+  const remainingSize = 2 ** 14 + 2 ** 13 - (obj.bytecode.length - 2) / 2;
```

Comment:
nit: why not use `bytecodeSize` here instead of retyping the math from the line before?